pkg/apb/bind_test.go

```diff
@@ -1,13 +1,13 @@
 package apb
 
 import (
-	"fmt"
 	"testing"
 
 	ft "github.com/fusor/ansible-service-broker/pkg/fusortest"
 )
 
-func TestDecodeOutput(t *testing.T) {
+func TestBind(t *testing.T) {
+	t.Skip("skipping bind until we can pass in a mock client")
```

Comment:
I'm purposely skipping this test because Bind uses a docker client that we need to mock. Would be nice to be able to pass in a mock client and set an expectation of what commands should get passed to runCommand, etc.